class ExpectedException(Exception):
    ...


class ExitRound(ExpectedException):
    pass


class ExitLoop(ExpectedException):
    pass
